src/platforms/windows/daemon/windowssplittunnel.cpp

```diff
@@ -8,15 +8,13 @@
 #include "windowsfirewall.h"
 #include "logger.h"
 
-#include <QNetworkInterface>
 #define PSAPI_VERSION 2
 #include <Windows.h>
 #include <psapi.h>
-#include <fwpmu.h>
-#include <initguid.h>
-#include <iphlpapi.h>
+
 #include <QCoreApplication>
 #include <QFileInfo>
+#include <QNetworkInterface>
```

Comment:
Please, check if there are implications in the removal of these 3 includes. It turned out that windot11.h is not compatible with the new msvc preprocessor.

Reply:
Seems to be fine to remove it :)